debug_module.tex

```diff
@@ -299,6 +299,13 @@ \section{System Bus Access} \label{systembusaccess}
 involving a hart, regardless of whether Program Buffer is implemented.
 The System Bus Access block uses physical addresses.
 
+If a hart has a cache, the data in the cache may not be consistent with the
+memory as accessed directly through the system bus. It is up to the debugger to
+have each hart write back its caches before performing a system bus read, and
+to invalidate the caches after a system bus write. The simplest way to do this
+is by executing {\tt fence} instructions using the Program Buffer. If there is
```

Comment:
"fence" instructions do not necessarily write back caches, or anything else of that nature. The RISC-V ISA doesn't even mention caches, so I would hesitate to mention them here. I agree we need some sort of statement like that, but I don't think we should claim here that a "fence" does anything specific with regards to caches.

Reply:
Good point. I've removed the reference to fence instructions.